### PR TITLE
Drugsfda

### PR DESCRIPTION
**Pull request by beardedfinch**

### CODE REVIEW

File: src/pages/apis/drug/drugsfda/_infographic_definitions.json [hunk 1]
```diff
@@ -0,0 +1,140 @@
+{
```
Comment (dkrylovsb) [inline src/pages/apis/drug/drugsfda/_infographic_definitions.json:1]: @beardedfinch This file refers to Enforcement in a number of places.

File: src/pages/apis/drug/drugsfda/_meta.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+documentTitle: openFDA › Drugs › Drugs@FDA
+label: Drugs@FDA
+title: Drugs@FDA
+description: "Drug products are identified and reported using a unique, three-segment number, called the National Drug Code (NDC), which serves as a universal product identifier for drugs."
```
Comment (dkrylovsb) [inline src/pages/apis/drug/drugsfda/_meta.yaml:4]: @beardedfinch This is a description from NDC.

File: src/pages/apis/drug/drugsfda/_infographic_definitions.json [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  "globalDefs": {
+    "startYear": 2012,
```
Comment (dkrylovsb) [inline src/pages/apis/drug/drugsfda/_infographic_definitions.json:3]: @beardedfinch There are submissions going all the way back to 1939-02-09.

File: src/pages/apis/drug/drugsfda/_meta.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+datasets:
+  - drugsfda
+path: /apis/drug/drugsfda
+api_path: /drug/drugsfda
+start: 20090101
```
Comment (dkrylovsb) [inline src/pages/apis/drug/drugsfda/_meta.yaml:9]: @beardedfinch The oldest drug submission is 19390209; we should probably adjust the `start` value, too.

File: src/pages/apis/drug/drugsfda/understanding-the-api-results/_examples.json [hunk 5]
```diff
@@ -0,0 +1,115 @@
+{
```
Comment (dkrylovsb) [inline src/pages/apis/drug/drugsfda/understanding-the-api-results/_examples.json:1]: This file still uses the old schema.

File: src/pages/apis/drug/drugsfda/_infographic_definitions.json (hunk 1 above)
Comment (beardedfinch) [reply src/pages/apis/drug/drugsfda/_infographic_definitions.json:1]: This file isn't used; I'll delete it.

File: src/pages/apis/drug/drugsfda/_meta.yaml (hunk 4 above)
Comment (beardedfinch) [reply src/pages/apis/drug/drugsfda/_meta.yaml:9]: Fixed.

File: src/pages/apis/drug/drugsfda/_meta.yaml (hunk 2 above)
Comment (beardedfinch) [reply src/pages/apis/drug/drugsfda/_meta.yaml:4]: Fixed.

File: src/pages/apis/drug/drugsfda/understanding-the-api-results/_examples.json (hunk 5 above)
Comment (beardedfinch) [reply src/pages/apis/drug/drugsfda/understanding-the-api-results/_examples.json:1]: Fixed.

**Pull request merged**